### DISCUSSION TITLE
kb/hardware/pcie

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/kb#6] [General]
# kb/hardware/pcie

Knowledge Base of @jiegec

https://jia.je/kb/hardware/pcie.html

**Comment by vickiegpt** [comment]
我现在的思考是，Intel规定 CXL.mem with Back Invalidation是device是没有计算能力的CHA，但是只有半功能的Snoopy fileter和DCOH，导致有两种Bias。Testing DCOH本来就是一个很tricky的work，不太像是马上能做出来的。我听vendor说，大部分未来的产品都是CXL.mem with Back Invalidation，因为cache在DCOH里有codesign可以做。
